### DISCUSSION TITLE
Recommended way to build a blog with Sulu 2.6?

### DISCUSSION THREAD

**Discussion by RafaelKr** [sulu/SuluArticleBundle#713] [Q&A] [answered]
We're using Sulu 2.6.

We'd like to build a blog with some posts. In the beginning there won't be many blog posts. What are our possibilities?
- Use Pages for now. Would it be possible to migrate them to articles later?
- Use ArticleBundle 3.0 in its current state. Is it stable enough yet?
- Any other recommended option?

**Comment by alexander-schranz** [comment, marked as answer]
> Use ArticleBundle 3.0 in its current state. Is it stable enough yet?

ArticleBundle 3.0 can only be used with Sulu 3.0. There are some devs already start new projects on top of 3.0 branch but then you need really keep yourself uptodate with subscribing thigns to keep. As there will still be lot of bc breaks at start of next year.

> Use Pages for now. Would it be possible to migrate them to articles later?

Only if you manually migrate it to articles. On 2.6 using document manager on 3.0 doctrine migrations or custom command. Both can be done aslong as you keep the template properties the same.

> Any other recommended option?

Recommended way for Blogs, News, ... is the ArticleBundle.

You prefer stable Version go with Sulu + Sulu Article Bundle 2.6 and migrate to Sulu 3.0 when released.

You want early adopt Sulu 3.x start with Sulu 3.x but keep yourself really uptodate about all discussions and changes by subscribing the sulu/sulu repository and update things as soon as possible.

---

Worth mentioning. @TheCadien did also create a SuluNewsBundle you find this and more bundles in https://github.com/friendsOfSulu which may be a alternative. How smooth the migration to Sulu 3.0 and its Article Bundle will be I'm not sure yet.

**Reply by RafaelKr** [reply]
Thanks! As we require a stable project we'll proceed with using pages and migrate them to the ArticleBundle entites after 3.0 is released.
